website/docs/reference/resource-properties/versions.md

```diff
@@ -65,3 +65,62 @@ Note that the value of `defined_in` and the `alias` configuration of a model are
 - Follow a consistent naming convention for model versions and aliases.
 - Use `defined_in` and `alias` only if you have good reason.
 - Create a view that always points to the latest version of your model. You can automate this for all versioned models in your project with an `on-run-end` hook. For more details, read the full docs on ["Model versions"](/docs/collaborate/govern/model-versions#configuring-database-location-with-alias)
+
+### Detecting breaking changes
+
+When you use the `state:modified` selection method in Slim CI, dbt will detect changes to model contracts, and raise an error if any of those changes could be breaking for downstream consumers.
```

Comment:
Let's add 'versioned' here to clarify that breaking change detection only occurs for contracted models that are also versioned: 

"dbt will detect changes to model contracts" -> "dbt will detect changes to _versioned_ model contracts"